### PR TITLE
Laravel 11.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 11.x.

**Before merging**, you need to:

- Checkout the `l11-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/rabbitevents.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "nuwber/rabbitevents": "dev-l11-compatibility",
    }
}
```

Finally, run: `composer update`

File: composer.json [hunk 1]
```diff
@@ -1,67 +1,75 @@
...
+        {
+            "name": "Eugene Kirdzei",
+            "email": "masterjus@gmail.com"
+        }
+    ],
+    "require": {
+        "php": "^8.1",
+        "ext-json": "*",
+        "enqueue/amqp-lib": "^0.10",
+        "illuminate/support": "^9.0|^10.0|^11.0",
```
Comment (eugene-nuwber) [inline composer.json:25]:
```suggestion
        "illuminate/support": "9.0 - 11.0",
```

**Comment by jasonmccreary** [comment]
It will be released soon. This is to help you and your users be ready ahead of time.

**Comment by eugene-nuwber** [comment]
@jasonmccreary yes, thank you

**Comment by eugene-nuwber** [comment]
@laravel-shift Unfortunately, to change the composer.json file isn't enough https://laravel.com/docs/11.x/upgrade#packages

**Comment by eugene-nuwber** [comment]
Done in #131

**Comment by eugene-nuwber** [comment]
Done in #131

**Pull request closed without merging**